db/migrate/20220119122040_add_withdrawn_or_declined_for_candidate_by_provider_to_application_choices.rb

```diff
@@ -0,0 +1,5 @@
+class AddWithdrawnOrDeclinedForCandidateByProviderToApplicationChoices < ActiveRecord::Migration[6.1]
+  def change
+    add_column :application_choices, :withdrawn_or_declined_for_candidate_by_provider, :boolean
```

Comment:
This is quite wordy as a db column name.

How about `withdrawn_on_behalf_of_candidate` ?

Reply:
The column is added in this [PR](https://github.com/DFE-Digital/apply-for-teacher-training/pull/6386). Agree it's verbose but we wanted to be specific that this flag is only set when the action is performed by the provider (as opposed to a support user/dev if that scenario ever occurs)

Reply:
It's also meant to cover `declined` on behalf of candidate as well